test/runner/requirements/constraints.txt

```diff
@@ -17,3 +17,4 @@ ntlm-auth >= 1.0.6 # message encryption support
 requests-ntlm >= 1.1.0 # message encryption support
 requests-credssp >= 0.1.0 # message encryption support
 voluptuous >= 0.11.0 # Schema recursion via Self
+botocore >= 1.10.30 # adds support for the following AWS services: secretsmanager, fms, acm-pca, neptune
```

Comment:
I think this affects all integration tests. Instead, I think it belongs in `integration.cloud.aws.txt`. @mattclay can you confirm?

Reply:
This is the correct place to put version constraints. It affects all tests, but will not require the package to be installed for all tests. It will, however, require the version constraint be met whenever the package is installed.

Package requirements (only the package name) should be in the appropriate requirements file, such as `integration.cloud.aws.txt`. However, package constraints (version number requirements) should be placed in `constraints.txt`. This avoids having conflicting version requirements across tests while still allowing different tests to require different packages.